diff --git a/mavros/src/lib/uas_tf.cpp b/mavros/src/lib/uas_tf.cpp
index 365d3773..0c8c74d3
--- a/mavros/src/lib/uas_tf.cpp
+++ b/mavros/src/lib/uas_tf.cpp
@@ -16,7 +16,7 @@
 #include <vector>
 
 #include "mavros/mavros_uas.hpp"
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 
 using namespace mavros::uas;  // NOLINT
 
diff --git a/mavros/src/plugins/global_position.cpp b/mavros/src/plugins/global_position.cpp
index 171e9311..29739ce2
--- a/mavros/src/plugins/global_position.cpp
+++ b/mavros/src/plugins/global_position.cpp
@@ -19,7 +19,7 @@
 #include <angles/angles.h>
 
 #include <string>
-#include <tf2_eigen/tf2_eigen.hpp>
+#include <tf2_eigen/tf2_eigen.h>
 #include <GeographicLib/Geocentric.hpp>     // NOLINT
 
 #include "rcpputils/asserts.hpp"
diff --git a/mavros/src/plugins/home_position.cpp b/mavros/src/plugins/home_position.cpp
index 57325fe6..861932ae
--- a/mavros/src/plugins/home_position.cpp
+++ b/mavros/src/plugins/home_position.cpp
@@ -17,7 +17,7 @@
  */
 
 #include <memory>
-#include <tf2_eigen/tf2_eigen.hpp>
+#include <tf2_eigen/tf2_eigen.h>
 
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
diff --git a/mavros/src/plugins/imu.cpp b/mavros/src/plugins/imu.cpp
index f8e46f36..18876efb
--- a/mavros/src/plugins/imu.cpp
+++ b/mavros/src/plugins/imu.cpp
@@ -16,7 +16,7 @@
 
 #include <cmath>
 #include <string>
-#include <tf2_eigen/tf2_eigen.hpp>
+#include <tf2_eigen/tf2_eigen.h>
 
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
diff --git a/mavros/src/plugins/local_position.cpp b/mavros/src/plugins/local_position.cpp
index afbf868c..78d5cecf
--- a/mavros/src/plugins/local_position.cpp
+++ b/mavros/src/plugins/local_position.cpp
@@ -17,7 +17,7 @@
  */
 
 #include <string>
-#include <tf2_eigen/tf2_eigen.hpp>
+#include <tf2_eigen/tf2_eigen.h>
 
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
diff --git a/mavros/src/plugins/setpoint_accel.cpp b/mavros/src/plugins/setpoint_accel.cpp
index e11ff474..db4b0351
--- a/mavros/src/plugins/setpoint_accel.cpp
+++ b/mavros/src/plugins/setpoint_accel.cpp
@@ -16,7 +16,7 @@
  * @{
  */
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/plugin.hpp"
diff --git a/mavros/src/plugins/setpoint_attitude.cpp b/mavros/src/plugins/setpoint_attitude.cpp
index 496c401f..d5f01df1
--- a/mavros/src/plugins/setpoint_attitude.cpp
+++ b/mavros/src/plugins/setpoint_attitude.cpp
@@ -22,7 +22,7 @@
 
 #include <memory>
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/plugin.hpp"
diff --git a/mavros/src/plugins/setpoint_position.cpp b/mavros/src/plugins/setpoint_position.cpp
index f1279d52..4adb340b
--- a/mavros/src/plugins/setpoint_position.cpp
+++ b/mavros/src/plugins/setpoint_position.cpp
@@ -16,7 +16,7 @@
 
 #include <GeographicLib/Geocentric.hpp>
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/plugin.hpp"
diff --git a/mavros/src/plugins/setpoint_raw.cpp b/mavros/src/plugins/setpoint_raw.cpp
index beb259fd..9e28f8e4
--- a/mavros/src/plugins/setpoint_raw.cpp
+++ b/mavros/src/plugins/setpoint_raw.cpp
@@ -16,7 +16,7 @@
 
 #include <algorithm>
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/plugin.hpp"
diff --git a/mavros/src/plugins/setpoint_trajectory.cpp b/mavros/src/plugins/setpoint_trajectory.cpp
index 732c0616..b1134147
--- a/mavros/src/plugins/setpoint_trajectory.cpp
+++ b/mavros/src/plugins/setpoint_trajectory.cpp
@@ -18,7 +18,7 @@
 #include <vector>
 #include <string>
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/plugin.hpp"
diff --git a/mavros/src/plugins/setpoint_velocity.cpp b/mavros/src/plugins/setpoint_velocity.cpp
index 9f18f5e4..53520a58
--- a/mavros/src/plugins/setpoint_velocity.cpp
+++ b/mavros/src/plugins/setpoint_velocity.cpp
@@ -16,7 +16,7 @@
  * @{
  */
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/plugin.hpp"
diff --git a/mavros/src/plugins/wind_estimation.cpp b/mavros/src/plugins/wind_estimation.cpp
index 659d7942..3905c8b9
--- a/mavros/src/plugins/wind_estimation.cpp
+++ b/mavros/src/plugins/wind_estimation.cpp
@@ -17,7 +17,7 @@
 
 #include <angles/angles.h>
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/plugin.hpp"
diff --git a/mavros_extras/src/plugins/camera.cpp b/mavros_extras/src/plugins/camera.cpp
index 5a78af87..a2d4747f
--- a/mavros_extras/src/plugins/camera.cpp
+++ b/mavros_extras/src/plugins/camera.cpp
@@ -14,7 +14,7 @@
  * @{
  */
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/plugin.hpp"
diff --git a/mavros_extras/src/plugins/distance_sensor.cpp b/mavros_extras/src/plugins/distance_sensor.cpp
index 21b6d236..bc88d829
--- a/mavros_extras/src/plugins/distance_sensor.cpp
+++ b/mavros_extras/src/plugins/distance_sensor.cpp
@@ -23,7 +23,7 @@
 #include <vector>
 #include <shared_mutex>     // NOLINT cpplint, that is almost 4 years since standard release!
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/plugin.hpp"
diff --git a/mavros_extras/src/plugins/fake_gps.cpp b/mavros_extras/src/plugins/fake_gps.cpp
index 2a5b99fb..227ee9ea
--- a/mavros_extras/src/plugins/fake_gps.cpp
+++ b/mavros_extras/src/plugins/fake_gps.cpp
@@ -20,7 +20,7 @@
 
 #include <string>
 
-#include <tf2_eigen/tf2_eigen.hpp>
+#include <tf2_eigen/tf2_eigen.h>
 #include <GeographicLib/Geocentric.hpp>
 #include <GeographicLib/Geoid.hpp>
 
diff --git a/mavros_extras/src/plugins/guided_target.cpp b/mavros_extras/src/plugins/guided_target.cpp
index c481e08d..2b6da16b
--- a/mavros_extras/src/plugins/guided_target.cpp
+++ b/mavros_extras/src/plugins/guided_target.cpp
@@ -18,7 +18,7 @@
 
 #include <GeographicLib/Geocentric.hpp>
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/plugin.hpp"
diff --git a/mavros_extras/src/plugins/hil.cpp b/mavros_extras/src/plugins/hil.cpp
index 31a30bfc..e5d569ed
--- a/mavros_extras/src/plugins/hil.cpp
+++ b/mavros_extras/src/plugins/hil.cpp
@@ -19,7 +19,7 @@
 
 #include <algorithm>
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/plugin.hpp"
diff --git a/mavros_extras/src/plugins/landing_target.cpp b/mavros_extras/src/plugins/landing_target.cpp
index 60106209..72a087c1
--- a/mavros_extras/src/plugins/landing_target.cpp
+++ b/mavros_extras/src/plugins/landing_target.cpp
@@ -17,7 +17,7 @@
 #include <algorithm>
 #include <string>
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/utils.hpp"
diff --git a/mavros_extras/src/plugins/mocap_pose_estimate.cpp b/mavros_extras/src/plugins/mocap_pose_estimate.cpp
index 4903e79a..a4be009f
--- a/mavros_extras/src/plugins/mocap_pose_estimate.cpp
+++ b/mavros_extras/src/plugins/mocap_pose_estimate.cpp
@@ -15,7 +15,7 @@
  * @{
  */
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/plugin.hpp"
diff --git a/mavros_extras/src/plugins/mount_control.cpp b/mavros_extras/src/plugins/mount_control.cpp
index 09075473..af9e89bc
--- a/mavros_extras/src/plugins/mount_control.cpp
+++ b/mavros_extras/src/plugins/mount_control.cpp
@@ -18,7 +18,7 @@
 
 #include <memory>
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/plugin.hpp"
diff --git a/mavros_extras/src/plugins/odom.cpp b/mavros_extras/src/plugins/odom.cpp
index 634166be..aa709cd7
--- a/mavros_extras/src/plugins/odom.cpp
+++ b/mavros_extras/src/plugins/odom.cpp
@@ -18,7 +18,7 @@
 
 #include <string>
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/plugin.hpp"
diff --git a/mavros_extras/src/plugins/vibration.cpp b/mavros_extras/src/plugins/vibration.cpp
index 73fbdab1..6d752315
--- a/mavros_extras/src/plugins/vibration.cpp
+++ b/mavros_extras/src/plugins/vibration.cpp
@@ -16,7 +16,7 @@
 
 #include <string>
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/plugin.hpp"
diff --git a/mavros_extras/src/plugins/vision_pose_estimate.cpp b/mavros_extras/src/plugins/vision_pose_estimate.cpp
index 8793b9ca..cfde5e3d
--- a/mavros_extras/src/plugins/vision_pose_estimate.cpp
+++ b/mavros_extras/src/plugins/vision_pose_estimate.cpp
@@ -17,7 +17,7 @@
 
 #include <string>
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/utils.hpp"
diff --git a/mavros_extras/src/plugins/wheel_odometry.cpp b/mavros_extras/src/plugins/wheel_odometry.cpp
index 6ea61754..c65a9624
--- a/mavros_extras/src/plugins/wheel_odometry.cpp
+++ b/mavros_extras/src/plugins/wheel_odometry.cpp
@@ -18,7 +18,7 @@
 #include <string>
 #include <vector>
 
-#include "tf2_eigen/tf2_eigen.hpp"
+#include "tf2_eigen/tf2_eigen.h"
 #include "rcpputils/asserts.hpp"
 #include "mavros/mavros_uas.hpp"
 #include "mavros/plugin.hpp"
